Cargo.toml

```diff
@@ -5,4 +5,6 @@ version = "1.0.0"
 edition = "2021"
 
 [dependencies]
+lazy_static = "1"
```

Comment:
This was necessary for the 1 -> 1 port in https://github.com/poweranalyses-org/rmathlib/blob/5668a11c4fee19a0c1ba30b0523c5e29bf838ee6/src/pgamma.rs#L61-L65

Do you want to keep it, @rikhuijzer?

Reply:
Yes let's keep and maybe later we can see whether we convert to a const manually. Would maybe be cool to be able to call this a zero-dependency crate.

Reply:
Ok removing, we are going for a zero dependency crate.